src/domain/schemas/room-schemas.js

```diff
@@ -2,6 +2,15 @@ import joi from 'joi';
 import { ROOM_USER_ROLE } from '../constants.js';
 import { emailSchema, idOrKeySchema, slugSchema } from './shared-schemas.js';
 
+export const getAllOrPostRoomMediaParamsSchema = joi.object({
```

Comment:
Tiny thing, I would vote for having two schemas here, even if they are the same, than one called `AOrB`.
If that's also fine with you.

Reply:
Totally fine, split into two it is :-)